### PR TITLE
[RFC] pnor: Increase PAYLOAD partition size

### PR DESCRIPTION
**Pull request by hegdevasant**

### CODE REVIEW

**Comment by hegdevasant** [comment]
I have created PR to see whether it passes CIs or not. Please do not merge this PR.

If everything goes fine, I will send PR to `pnor` project itself. 

-Vasant

**Comment by dcrowell77** [comment]
@anoo1 - Is there a corresponding BMC change required?

**Comment by anoo1** [comment]
i think it's automatically, but checking to make sure. Also you might want to update the comment that has the size hardcoded to reflect the new size.

File: openpower/package/openpower-pnor-p10/0001-P10-Increase-PAYLOAD-partition-size-to-1MB.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ p10Layouts/defaultPnorLayout_64.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/p10Layouts/defaultPnorLayout_64.xml b/p10Layouts/defaultPnorLayout_64.xml
+index c2ebc14..9d75f08 100644
+--- a/p10Layouts/defaultPnorLayout_64.xml
++++ b/p10Layouts/defaultPnorLayout_64.xml
+@@ -173,7 +173,7 @@ Layout Description
+     <section>
+         <description>Payload (512KiB)</description>
```
Comment (anoo1) [inline openpower/package/openpower-pnor-p10/0001-P10-Increase-PAYLOAD-partition-size-to-1MB.patch:20]: The Table of Contents will be automatically updated, so this change doesn't have any other dependencies. Just need to update the description with the new size.

Comment (hegdevasant) [reply openpower/package/openpower-pnor-p10/0001-P10-Increase-PAYLOAD-partition-size-to-1MB.patch:20]: Thanks @anoo1 . We will push changes to pnor repository. 

-Vasant

**Comment by hegdevasant** [comment]
I posted patch to pnor repository in GHE.. which will be replicated to open-power/pnor repo soon.

Hence closing this PR.

-Vasant

**Pull request closed without merging**